app/views/api/v1/work/_work.json.jbuilder

```diff
@@ -113,7 +113,13 @@ if valid_json?(related_identifier)
   end
 end
 
-bool =  Hyrax::WorkShowPresenter.new(SolrDocument.new(work, ''), nil).representative_presenter&.solr_document&.visibility == "open"
+current_account = Account.find_by(tenant: Apartment::Tenant.current)
+
+if current_account.cname == 'oar.bl.uk'
+  bool = false
```

Comment:
Let's try `true` here instead of false.  It might let in some blue icons but that might be safer than not showing any thumbnails at all.  Not sure if they are used at all in the frontend but maybe.
```suggestion
  bool = true
```